.github/workflows/develop.yml

```
@@ -20,8 +20,18 @@ jobs:
                     test/end-to-end-tests/logs/**/*
                     test/end-to-end-tests/synapse/installations/consent/homeserver.log
                 retention-days: 14
-            - name: Archive performance benchmark
-              uses: actions/upload-artifact@v2
+            - name: Download previous benchmark data
+              uses: actions/cache@v1
+              with:
+                path: ./cache
+                key: ${{ runner.os }}-benchmark
+            - name: Store benchmark result
+              uses: matrix-org/github-action-benchmark@jsperfentry
               with:
-                name: performance-entries.json
-                path: test/end-to-end-tests/performance-entries.json
+                tool: 'jsperformanceentry'
+                output-file-path: test/end-to-end-tests/performance-entries.json
+                external-data-json-path: ./cache/benchmark-data-template.json
+                fail-on-alert: false
+            - name: Push benchmark result
+              if: ${{ github.ref == 'refs/heads/develop' }}
+              run: git push 'https://matrixbot:${{ secrets.DEPLOY_GH_PAGES }}@github.com/matrix-org/matrix-react-sdk.git' gh-pages:gh-pages
```

@RiotRobot is probably a closer match for our needs. I'll update the secret.

```suggestion
              run: git push 'https://RiotRobot:${{ secrets.DEPLOY_GH_PAGES }}@github.com/matrix-org/matrix-react-sdk.git' gh-pages:gh-pages
```